go/vt/vtgate/vindexes/consistent_lookup_test.go

```diff
@@ -55,22 +109,6 @@ func TestConsistentLookupInit(t *testing.T) {
 	}
 }
 
-func TestConsistentLookupInfo(t *testing.T) {
```

Comment:
`Test[VindexType]Info()` tests are now handled in `Test[VindexType]CreateVindex` tests, are redundant and are removed in this PR.